spec.bs

```diff
@@ -1324,7 +1324,7 @@ To <dfn>parse a pattern string</dfn> given a [=/pattern string=] |input|, [=/opt
         </dl>
       </div>
     1. If |open token| is not null:
-      1. Set |prefix| be the result of running [=consume text=] given |parser|.
+      1. Let |prefix| be the result of running [=consume text=] given |parser|.
       1. Set |name token| to the result of running [=try to consume a token=] given |parser| and "<a for=token/type>`name`</a>".
       1. Set |regexp or wildcard token| to the result of running [=try to consume a regexp or wildcard token=] given |parser| and |name token|.
       1. Let |suffix| be the result of running [=consume text=] given |parser|.
```

Comment:
I can't comment on the exact line, but I believe step 6 below should also be changed to "Let":

>       1. Set |modifier token| to the result of running [=try to consume a modifier token=] given |parser|.

Reply:
Yeah this looks also a typo.